app/views/application/_menu.html.haml

```diff
@@ -0,0 +1,3 @@
+%ul.menu
+  %li{class: ('active' if current_page?(controller: 'books'))}= link_to 'Books', books_path
+  %li{class: ('active' if current_page?(controller: 'authors'))}= link_to 'Authors_path', authors_path
```

Comment:
Files should end with a trailing newline<br>Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace<br>Line is too long. [102/80]